### ISSUE TITLE
ansible-galaxy still installs v1.1.1

### ISSUE THREAD

**Issue by luka5z** [angstwad/docker.ubuntu#93]
Why `ansible-galaxy` still installs v1.1.1, whereas from list of project's tag it looks like it should fetch 1.1.4?

```
$ ansible-galaxy install angstwad.docker_ubuntu 
- downloading role 'docker_ubuntu', owned by angstwad
- downloading role from https://github.com/angstwad/docker.ubuntu/archive/v1.1.1.tar.gz
...
```

**Comment by angstwad** [comment]
Not sure why it's doing this, but it is.  I've manually updated in Galaxy several times to no effect.

In the meantime, let's assume the best installation method is to install from git:

```
ansible-galaxy install git+https://github.com/angstwad/docker.ubuntu
```

**Comment by luka5z** [comment]
I suspect it is due to the fact that ansible-galaxy takes latest version based on alphabetical order, exactly like Github tags are sorted.

**Comment by gotgenes** [comment]
I believe @luka5z is correct. The tags need to be consistent; that is, it should be published as "v1.1.15" instead of "1.1.15"

```
$ python
Python 2.7.11 (default, Jan 22 2016, 08:29:18)
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 'v1.1.11' > '1.1.15'
True
>>> 'v' > '1'
True
>>>
```

**Comment by angstwad** [comment]
This is fixed with the v1.1.6 and I'll get around to re-tagging in the future sometime.

**Comment by gotgenes** [comment]
This is resolved now.

I'll note that from the discussion in [this Stack Overflow question](http://stackoverflow.com/questions/2006265/is-there-a-standard-naming-convention-for-git-tags), as well as [this Stack Overflow question](http://stackoverflow.com/questions/21639437/git-flow-release-branches-and-tags-with-or-without-v-prefix) about the v-prefix. While using a v-prefix for VCS tags is accepted (see [SemVer 1.0.0 on SemVerTag](http://semver.org/spec/v1.0.0.html#tagging-specification-semvertag), in SemVer 2.0.0, that recommendation was removed. This is touched on in [the FAQ of a yet unreleased version of SemVer spec](https://github.com/mojombo/semver/pull/234).

So docker.ubuntu should get released without `v` prefixes, but if you want to use v-prefixes in your Git tags, that's okay, but an outdated practice.

Technically you were doing the right thing by making releases without the v-prefix. You'll have to correct the old release versions to drop the v-prefix to get `ansible-galaxy` to cooperate with you, though.
